client/src/components/RegistrationInfo.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import {Col, Row} from "react-materialize";
+
+class RegistrationInfo extends Component{
```

Comment:
If your component just renders some markup, better to use a function declaration. [Style Guide](https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless)

Reply:
I dont see here problems. Like Serhiy said that must be separate state page.

Reply:
You don't understand me, I told about function declaration of a component. Please follow the link